### ISSUE TITLE
Unknown provider: AuthProvider <- Auth

### ISSUE THREAD

**Issue by tjslater** [bistel-a3/generator-a3#1]
Upon scaffolding, and running grunt build and grunt serve, this javascript error shows up in the console.

Uncaught Error: [$injector:unpr] Unknown provider: AuthProvider <- Auth

I'm somewhat new to this scaffold, so I couldn't immediately find where the issue is coming from.

**Comment by kiwikern** [comment]
I'm running into a similar problem right after scaffolding:
`Error: [$injector:modulerr] Failed to instantiate module a3.common due to:
Error: [$injector:nomod] Module 'a3.common' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.`

Have you found a solution already?
To me, it seems like some modules haven't been referenced in index.html, e.g. 
    `<script src="common/common.module.js"></script>` is missing.

**Comment by tjslater** [comment]
No, I didn't, unfortunately. It looks like a similar error, though an Auth module usually sits apart on its own. Though it's entirely possible to bring it in under common.
